signer/src/transaction_signer.rs

```diff
@@ -488,10 +499,17 @@ where
                     self.signer_private_key,
                 )?;
                 self.wsts_state_machines.insert(msg.txid, state_machine);
+                // Let's give the others some slack
+                tracing::debug!(
+                    "Sleeping a bit to give the other peers some slack to get DkgBegin"
+                );
+                tokio::time::sleep(Duration::from_secs(3)).await;
+
```

Comment:
Oh this is a band-aid until we revamp DKG?

Reply:
Correct -- we bumped this to 10s in the current deployment and everyone got their DKG shares